### DISCUSSION TITLE
Resend Email on failure after configured delay

### DISCUSSION THREAD

**Discussion by ptrxwsmitt** [pretix/pretix#3151] [Ideas]
We are often facing the issue, that we hit the hourly/daily limits of our SMTP provider for sending out Mails.
Therefore we thought it would be great to have some kind of retry-feature for failed sending of Mails.
Alternative solutions:
- Send Queue with configurable limits per hour/day
- Filters and manual resend options for failed mail

I'd be happy also hear other possible solutions (aside renting a send-mail service with an smtp relay :D).

**Comment by raphaelm** [comment]
pretix *does* retry emails many times to to a total delay of 1h. However it does so only if the SMTP error code does look like a temporary failure.

Anything more (longer intervals, better insights to it) would be nice but would require a pretty deep rewrite of the email stack.
